src/transformers/benchmark/benchmark_args_utils.py

```diff
@@ -57,22 +57,22 @@ class BenchmarkArguments:
         metadata={"help": "List of sequence lengths for which memory and time performance will be evaluated"},
     )
 
-    no_inference: bool = field(default=False, metadata={"help": "Don't benchmark inference of model"})
-    no_cuda: bool = field(default=False, metadata={"help": "Whether to run on available cuda devices"})
-    no_tpu: bool = field(default=False, metadata={"help": "Whether to run on available tpu devices"})
+    inference: bool = field(default=True, metadata={"help": "Whether to benchmark inference of model"})
```

Comment:
I like this text! Could you add the following sentence to all `default=True` args: "inference can be disabled via `--no-inference.`" So that people know how the param can be set to `False` via the command line